integration-cli/docker_cli_start_volume_driver_unix_test.go

```diff
@@ -119,11 +119,6 @@ func (s *DockerExternalVolumeSuite) SetUpSuite(c *check.C) {
 			http.Error(w, err.Error(), 500)
 		}
 
-		p := hostVolumePath(pr.name)
-		if err := os.RemoveAll(p); err != nil {
-			http.Error(w, err.Error(), 500)
-		}
-
 		w.Header().Set("Content-Type", "application/vnd.docker.plugins.v1+json")
```

Comment:
Hmm, I thought the API version would get bumped whenever *any* of the API changed. (The volume API is the only one in stable at the moment, but when networking lands they would have to "share" this version number.)

If that's not the case we should talk about introducing per subsystem versioning now I think.